config/katello.migrations/160503102427-gutterball.rb

```diff
@@ -0,0 +1 @@
+answers['katello::plugin::gutterball'].delete if answers['katello::plugin::gutterball']
```

Comment:
Was trying this out and hit this error:
`/etc/foreman-installer/scenarios.d/katello.migrations/160503102427-gutterball.rb:1:in `delete': wrong number of arguments (0 for 1) (ArgumentError)`

I think it needs to be `answers.delete("katello::plugin::gutterball")`  (and same on the other migrations in this pr)